### PR TITLE
Actually fix url fragments not coming into view

### PR DESCRIPTION
**Pull request by diogoscf**
@aadibajpai ;)

### CODE REVIEW

**Comment by aadibajpai** [comment]
@gfting I'ma merge and redeploy we can't get any worse lol

**Comment by aadibajpai** [comment]
thanks @diogoscf <3

File: public/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class='title'>The VandyHacks: Retro Edition Hacker Guide</h1>
     <script>
         window.onload = () => {
             if(window.location.hash) {
-                window.location.hash.scrollIntoView();
+                document.getElementsByName(window.location.hash.replace("#", ""))[0].scrollIntoView();
```
Comment (gfting) [inline public/index.html:18]: Nice! Why not just getElementByName though?

Comment (diogoscf) [reply public/index.html:18]: because that's not a thing?

Comment (gfting) [reply public/index.html:18]: lmao thanks

Comment (diogoscf) [reply public/index.html:18]: https://stackoverflow.com/a/7816899/7729429

**Pull request merged**